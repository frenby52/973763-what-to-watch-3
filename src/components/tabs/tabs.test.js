import React from "react";
import renderer from "react-test-renderer";
import Tabs from "./tabs";

it(`Tabs component should render correct`, () => {
  const tree = renderer
    .create(<Tabs onTabClick={() => {}} activeTab={0} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
