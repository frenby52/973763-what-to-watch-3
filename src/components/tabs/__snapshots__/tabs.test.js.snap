// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs component should render correct 1`] = `
<ul
  className="movie-nav__list"
>
  <li
    className="movie-nav__item movie-nav__item--active"
  >
    <a
      className="movie-nav__link"
      href="#"
      onClick={[Function]}
    >
      Overview
    </a>
  </li>
  <li
    className="movie-nav__item "
  >
    <a
      className="movie-nav__link"
      href="#"
      onClick={[Function]}
    >
      Details
    </a>
  </li>
  <li
    className="movie-nav__item "
  >
    <a
      className="movie-nav__link"
      href="#"
      onClick={[Function]}
    >
      Reviews
    </a>
  </li>
</ul>
`;
